prisma/migrations/20221206162122_init/migration.sql

```diff
@@ -1,20 +1,20 @@
-/*
-  Warnings:
-
-  - You are about to drop the `Post` table. If the table is not empty, all the data it contains will be lost.
-
-*/
--- DropForeignKey
-ALTER TABLE "Post" DROP CONSTRAINT "Post_authorId_fkey";
+-- CreateTable
+CREATE TABLE "User" (
+    "id" SERIAL NOT NULL,
+    "name" TEXT NOT NULL,
+    "telegramId" TEXT,
+    "telegramUserName" TEXT NOT NULL,
+    "telegramDpUrl" TEXT,
```

Comment:
Do you think it would be good to have uniqueness constraint on telegramId? 

Also I think maybe we could do Varchar(32) on usernames since that is telegram's constraint on usernames as well.